packages/melody-plugin-jsx/src/helpers/index.js

```diff
@@ -88,7 +88,7 @@ export function assembleNode(path, expression) {
             if (is(expression, 'StringLiteral')) {
                 return t.jSXText(expression.value);
             }
-            if (is(expression, 'Expression')) {
+            if (is(expression, 'Expression') && !t.isJSXElement(expression)) {
```

Comment:
why was it necessary btw?

Reply:
Yes. In babel-types `JSXElement` comes under `expression` so it will add `{}` like this 

```
 {<span> be </span>}
```